web_responsive/static/src/css/web_responsive.scss

```
@@ -160,6 +191,13 @@ $chatter_zone_width: 35%;
             display: flex;
             flex-direction: column;
             justify-content: flex-start;
+            background: none;
+            img {
+                box-shadow: none;
+                transition: 300ms ease;
+                transition-property: box-shadow, transform;
+                transition-property: transform, box-shadow;
```

This seems duplicated. Just remove one of them:

```suggestion
                transition-property: box-shadow, transform;
```